pydjamodb/connection.py

```diff
@@ -71,6 +71,8 @@ def create_table(self,
             settings.PYDJAMODB_DATABASE.get('POINT_IN_TIME_RECOVERY', False) if set_point_in_time_recovery is None
             else set_point_in_time_recovery
         )
+        if not stream_specification and settings.PYDJAMODB_DATABASE.get('STREAM_ENABLED'):
+            stream_specification = {'stream_enabled': True, 'stream_view_type': 'NEW_AND_OLD_IMAGES'}
```

Comment:
We don't need old images. Use `NEW_IMAGE` only.

Reply:
ok, changed